apps/dashboard/src/utils/routes.ts

```diff
@@ -29,6 +29,9 @@ export const ROUTES = {
   INTEGRATIONS_UPDATE: '/integrations/:integrationId/update',
   API_KEYS: '/env/:environmentSlug/api-keys',
   ACTIVITY_FEED: '/env/:environmentSlug/activity-feed',
+  TEMPLATE_STORE: '/env/:environmentSlug/workflows/templates',
```

Comment:
Nit-picky: 

```suggestion
  TEMPLATE_STORE: '/env/:environmentSlug/workflow_templates',
```

Technically this is a different resource than `workflow`

Reply:
Will convert it to `workflow-templates` for consistency with other 2 word pages. altho I think it's cleaner than workflow-create and and workflow-templates, but not strong opinion on this

Reply:
Nope, unfortuently it's not possible :( as we want this modal to open on the "/workflows" page it has to be nested other-wise react-router throws an error for this. Moving this outside of the nested route, will open the modal outside of the context of the workflows page (blank screen). 

I will leave it as-is if this is very critical. We can relocate the editor to be under /workflows/editor/slug_id to allow a more dynamic namespace, but I don't think it's needed.

Reply:
🤦 It's fine. Leave it as is. It's a bit weird that `/templates` can be interpreted as a workflowID at the router level.